third_party/conan/docker/Dockerfile.clang7_opengl_qt

```diff
@@ -1,102 +1,33 @@
-       libbz2-dev=1.* \
-       libsqlite3-dev=3.* \
-       libreadline-dev=8.* \
-       xz-utils=5.* \
-       curl=7.* \
-       libncurses5-dev=6.* \
-       libncursesw5-dev=6.* \
-       liblzma-dev=5.* \
-       libstdc++-9-dev \
-       ca-certificates \
-       libglu1-mesa-dev \
-       mesa-common-dev \
-       libxmu-dev \
-       libxi-dev \
-       qt5-default \
-       libqt5webchannel5-dev \
-       libqt5websockets5-dev \
-       qtwebengine5-dev \
-       jq \
-       python2.7 \
-       zip \
-    && update-alternatives --install /usr/bin/clang++ clang++ /usr/bin/clang++-7 100 \
-    && update-alternatives --install /usr/bin/clang clang /usr/bin/clang-7 100 \
-    && update-alternatives --install /usr/bin/cc cc /usr/bin/clang-7 100 \
-    && update-alternatives --install /usr/bin/c++ c++ /usr/bin/clang++-7 100 \
-    && update-alternatives --install /usr/bin/cpp cpp /usr/bin/clang++-7 100 \
-    && ln -s /usr/include/locale.h /usr/include/xlocale.h \
-    && rm -rf /var/lib/apt/lists/* \
-    && wget -q --no-check-certificate https://cmake.org/files/v3.16/cmake-3.16.4-Linux-x86_64.tar.gz \
-    && tar -xzf cmake-3.16.4-Linux-x86_64.tar.gz \
-       --exclude=bin/cmake-gui \
-       --exclude=doc/cmake \
-       --exclude=share/cmake-3.12/Help \
-    && cp -fR cmake-3.16.4-Linux-x86_64/* /usr \
-    && rm -rf cmake-3.16.4-Linux-x86_64 \
-    && rm cmake-3.16.4-Linux-x86_64.tar.gz \
-    && groupadd 1001 -g 1001 \
-    && groupadd 1000 -g 1000 \
-    && groupadd 2000 -g 2000 \
-    && groupadd 999 -g 999 \
-    && useradd -ms /bin/bash conan -g 1001 -G 1000,2000,999 \
-    && printf "conan:conan" | chpasswd \
-    && adduser conan sudo \
-    && printf "conan ALL= NOPASSWD: ALL\\n" >> /etc/sudoers \
-    && wget --no-check-certificate --quiet -O /tmp/pyenv-installer https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer \
-    && chmod +x /tmp/pyenv-installer \
-    && /tmp/pyenv-installer \
-    && rm /tmp/pyenv-installer \
-    && update-alternatives --install /usr/bin/pyenv pyenv /opt/pyenv/bin/pyenv 100 \
-    && PYTHON_CONFIGURE_OPTS="--enable-shared" pyenv install 3.7.5 \
-    && pyenv global 3.7.5 \
-    && pip install -q --upgrade --no-cache-dir pip \
-    && pip install -q --no-cache-dir conan conan-package-tools \
-    && chown -R conan:1001 /opt/pyenv \
-    # remove all __pycache__ directories created by pyenv
-    && find /opt/pyenv -iname __pycache__ -print0 | xargs -0 rm -rf \
-    && update-alternatives --install /usr/bin/python python /opt/pyenv/shims/python 100 \
-    && update-alternatives --install /usr/bin/python3 python3 /opt/pyenv/shims/python3 100 \
-    && update-alternatives --install /usr/bin/pip pip /opt/pyenv/shims/pip 100 \
-    && update-alternatives --install /usr/bin/pip3 pip3 /opt/pyenv/shims/pip3 100
+RUN useradd -Ums /bin/bash conan && \
+    apt-get -qq update && \
+    DEBIAN_FRONTEND=noninteractive \
+        apt-get install -y --no-install-recommends \
```

Comment:
Super Nit: "--yes" is more readable for readers not too familiar.

If you decide to do that change, also consider the other docker files.

Reply:
Yes, I agree. But this time I will refrain from doing it because it's a lot of work since I have to rebuild all docker images afterwards and I feel this is not justified.

Thanks anyway, appreciated!